strawberryfields/gbs/plot.py

```diff
@@ -108,13 +112,14 @@ def _edge_coords(graph: nx.Graph, l: dict) -> dict:
 subgraph_node_size = 16
 
 
-def plot_graph(graph: nx.Graph, subgraph: Optional[list] = None,
-               size: int = 500) -> None:  # pragma: no cover
-    """Creates a plotly plot of the input graph.
+def plot_graph(
+    graph: nx.Graph, subgraph: Optional[list] = None, plot_size: int = 500
```

Comment:
Am happy with the `plot_size` change, but it should be updated in the docstring "Args" section